tests/subsys/pm/device_wakeup_api/src/main.c

```diff
@@ -13,9 +13,106 @@
 
 
 static const struct device *dev;
+static const struct device *devA;
+static const struct device *devC;
 static uint8_t sleep_count;
 
 
+/*
```

Comment:
note: this test is not really about the wakeup API, it's related to the system PM, better not mix test categories

Reply:
I plan to rename this test to be honest. I think we should avoid having multiple small applications to test only thing we can one application to test multiple things.

Reply:
anyway, will split this one, since both depends on the system PM is hard to de-couple them  keeping it simple to understand.